File: Prodavnica-UI/my-app/src/models/Korisnik.js
export class RegistracijaModel {
    constructor(
      Id,
      KorisnickoIme,
      Email,
      Sifra,
      PonovljenaSifra,
      Ime,
      Prezime,
      DatumRodjenja,
      Adresa,
      Slika,
      Tip
    ) 
    {
      this.Id = Id;
      this.KorisnickoIme = KorisnickoIme;
      this.Email = Email;
      this.Sifra = Sifra;
      this.PonovljenaSifra = PonovljenaSifra;
      this.Ime = Ime;
      this.Prezime = Prezime;
      this.DatumRodjenja = DatumRodjenja;
      this.Adresa = Adresa;
      this.Slika = Slika;
      this.Tip = Tip;
    }
  }

File: Prodavnica/Prodavnica/Models/Korisnik.cs
using System.ComponentModel.DataAnnotations;
using Prodavnica.Common;
namespace Prodavnica.Models
{
	public class Korisnik
	{
		public int Id { get; set; }

		[Required, RegularExpression(@"^[a-zA-Z0-9_]$")]
		public string? KorisnickoIme { get; set; }

		[Required, EmailAddress]
		public string? Email { get; set; }

		[Required]
		public string? Password { get; set; }

		[Required, MaxLength(35)]
		public string? Ime { get; set; }

		[Required, MaxLength(35)]
		public string? Prezime { get; set; }

		[Required, DataType(DataType.Date)]
		public DateTime DatumRodjenja { get; set; }

		[Required, MaxLength(30)]
		public string? Adresa { get; set; }

		public byte[]? Slika { get; set; }

		public ETipKorisnika Tip { get; set; }

		public EStatusVerifikacije Verifikacija { get; set; }

		public List<Porudzbina>? Porudzbinas { get; set; }

		public List<Proizvod>? Proizvods { get; set; }
	}
}
